File: src/test/java/Test/AndroidTests/LoginAndRegTests/Login_test.java
package Test.AndroidTests.LoginAndRegTests;

import AppiumSupport.BaseTestClass;

import org.junit.Test;

public class Login_test extends BaseTestClass {

    @Test
    public void correctLoginTest() {
        onboardingInterface.waitThenCloseOnBoardingPage();
        loginInterface.loginAsTester9();
        tapeInterface.closeTooltips();

    }

    @Test
    public void incorrectPhoneNumber() {
        onboardingInterface.waitThenCloseOnBoardingPage();
        loginInterface.enterIncorrectPhoneNumber();
        loginInterface.clickReceiveCodeBtn();
        loginInterface.waitErrorPhoneFormat();
        loginInterface.assertErrorPhoneFormat();
    }

    @Test
    public void incorrectCode() {
        onboardingInterface.waitThenCloseOnBoardingPage();
        loginInterface.enterPhoneNumber("9999999999");
        loginInterface.clickReceiveCodeBtn();
        loginInterface.waitCodeScreen();
        loginInterface.enterIncorrectCode();
        loginInterface.clickConfirmCodeBtn();
        loginInterface.waitErrorCode();
        loginInterface.assertErrorCode();
    }
}

File: src/test/java/Test/AndroidTests/LoginAndRegTests/Login_test.kt
package Test.AndroidTests.LoginAndRegTests

import AppiumSupport.BaseTestClass
import Pages.AndroidPages.LoginAndRegistration.User
import io.appium.java_client.pagefactory.AndroidBy

import org.junit.Before

import org.junit.Test

class Login_test : BaseTestClass() {


 @AndroidBy
    @Test
    fun correctLoginTest() {
        onboardingInterface.waitThenCloseOnBoardingPage()
        loginInterface.login(User.User1())
        tapeInterface.closeTooltips()
    }
    @AndroidBy
    @Test
    fun incorrectPhoneNumber() {
        onboardingInterface.waitThenCloseOnBoardingPage()
        loginInterface.enterIncorrectPhoneNumber()
        loginInterface.clickReceiveCodeBtn()
        loginInterface.waitErrorPhoneFormat()
        loginInterface.assertErrorPhoneFormat()
    }
    @AndroidBy
    @Test
    fun incorrectCode() {
        onboardingInterface.waitThenCloseOnBoardingPage()
        loginInterface.enterPhoneNumber("9999999999")
        loginInterface.clickReceiveCodeBtn()
        loginInterface.waitCodeScreen()
        loginInterface.enterIncorrectCode()
        loginInterface.clickConfirmCodeBtn()
        loginInterface.waitErrorCode()
        loginInterface.assertErrorCode()
    }
}